sdk/client/src/listeners/network-listener/utils/models.ts

```diff
@@ -24,3 +24,13 @@ export interface RequestResponsePair {
 	/** Whether this URL matched a `urlToBlock` so the contents should not be recorded. */
 	urlBlocked: boolean
 }
+
+export interface WebSocketEvent {
+	socketId: string
+	initiatorType: 'WebSocket'
+	type: 'open' | 'error' | 'close' | 'received' | 'sent'
+	name: string
+	startTime: number
```

Comment:
Called this `startTime` to avoid needing to patch the frontend to handle when sorting, but technically there is no endTime (i.e. this is a timeStamp)